hal/Cargo.toml

```diff
@@ -16,13 +16,13 @@ keywords = [
     "hal"
 ]
 license = "0BSD"
-repository = "https://github.com/michalfita/atsams-rust"
+repository = "https://github.com/atsams-rs/atsamx7x-rust"
 readme = "README.md"
-edition = "2018"
+edition = "2021"
```

Comment:
Are you 100% sure about that, Edition 2021 reached stability in 1.56 and somewhere earlier the Rust version was set to 1.46.

I don't mind setting minimum supported compiler @ 1.56, but I need to know there's commitment to do so.

Reply:
`rust_version = "1.46.0"`, set in the PACs define the minimum supported version. Edition is also crate-local. I do not see any issues using the latest edition in the HAL.